### PR TITLE
Fixes #12346: Update the techniques tests

### PR DESCRIPTION
**Pull request by Fdall**
https://www.rudder-project.org/redmine/issues/12346

### CODE REVIEW

**Comment by Fdall** [comment]
Deleted files which were just copies of v1.0 techniques and so not adapted.
Changed names to make possible test on multi versions techniques.
Some tests are still broken because of their conception (keyValue for exemple which use 2 metadatas and only 1 json, it should use 2 metadatas and 2 json operating on separated files).

**Comment by ncharles** [comment]
Does that means that tests on checkGenericFileContent are working ?

**Comment by Fdall** [comment]
I did not check for tests rightfulness but just for their format.
They are fine on my computer (failing on the initialisation of the 2nd test, meaning the proper execution of the test framework.

However I have a typo error in my commit. Will put a comment and amend.

File: techniques/fileDistribution/checkGenericFileContent/7.0/tests/simple_content_enforce.metadata [hunk 1]
```diff
@@ -1,5 +1,8 @@
-{
-  "directive": "simple_content_enforce.json",
-  "check": "simple_content_enforce.rb", 
+[{
+  "inits": [],
+  "directives": "[simple_content_enforce.json"],
```
Comment (Fdall) [inline techniques/fileDistribution/checkGenericFileContent/7.0/tests/simple_content_enforce.metadata]: Typo error

**Comment by Fdall** [comment]
Commit modified

**Pull request merged**